test/index.js

```diff
@@ -3,6 +3,7 @@
 import test from 'tape';
 import irecord from '../index.js';
 import rx from './rx';
+import stampit from './stampit';
```

Comment:
OH, I see what you mean! I haven't been getting enough sleep. ;) 

Let's just use require-all.